ui/src/components/Property/TagEditor.vue

```diff
@@ -0,0 +1,76 @@
+<script setup>
```

Comment:
Added files should contain the license header

Reply:
ok